vendor/arbiter/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ target_compile_definitions(${PDAL_ARBITER_LIB_NAME}
         ARBITER_CURL)
 
 if (MSVC)
-    find_library(SHLWAPI Shlwapi.lib)
+    set(SHLWAPI ShLwApi)
```

Comment:
This work in the official repo, why do we need to change it?

Reply:
I tried this with official repo as well it didn't worked.
It cannot find shlwapi lib.